src/schema-analyzer.ts

```diff
@@ -512,6 +517,13 @@ export class SchemaAnalyzer {
     }
   }
 
+  allowAbort() {
+    // Allow aborting the analysis.
+    if (this.fieldAndTypeAnalysisCounter++ % ALLOW_ABORT_INTERVAL_COUNT === 0) {
+      allowAbort();
```

Comment:
I don't get it?

Reply:
Yeah I couldn't decide what to name it, I do think having them named differently helps though. I'll update it to be unique.